src/asmcnc/keyboard/custom_keyboard.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, **kwargs):
         reload(sys)
         sys.setdefaultencoding('utf-8')
 
+        self.bind(layout=self.set_keyboard_background)
+        self.bind(do_translation=self.set_keyboard_background)
+
         self.text_instance = None
 
         self.kr_layout = "./asmcnc/keyboard/layouts/kr.json"
```

Comment:
Change these paths to be relative to current file. Otherwise unit tests all fail again.